_pages/docs/next-ls/editors.md

```diff
@@ -260,17 +260,25 @@ config = { experimental = { completions = { enable = true } } }
 
 ## Zed
 
-Zed will install and manage Next LS for you.
+Zed will install and manage Next LS for you. You can make `next-ls` the default with the following config:
 
 ```json
 {
-  // how you enable Next LS, notice the kebab casing
   "languages": {
     "Elixir": {
-      "language_servers": ["next-ls"]
+      "language_servers": ["!lexical", "!elixir-ls", "next-ls"]
+    },
+    "HEEX": {
+      "language_servers": ["lexical", "!elixir-ls", "next-ls"]
```

Comment:
i assume this needs to be `!lexical` ?

Reply:
Yep, fixed in [force push](https://github.com/elixir-tools/elixir-tools.dev/compare/7dd739ae3d6cac88ab04b8e695ae6974407bb804..41a013d8f2c139353f54c50b2d5d387ebd35d42e).